sig-instrumentation/annual-report-2022.md

```diff
@@ -4,118 +4,167 @@
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - We have a mentorship program and now have enrolled mentees/mentors.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes: Google, Red Hat, Sony, VMware, Intel, independent contributors, and more
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+    - SIG leads performed a [staffing and gap analysis](https://docs.google.com/document/d/1qeoP6i7GBTVJuJE1AGY5iU9dqmAOxrjqkfNQ2-rBeyI/edit#heading=h.849b7ydpl7ip) for the SIG's projects. We definitely can use more help, and need more part-time/full-time contributors.
 
-## Membership
+### Summary
 
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-- Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-- Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+- **KTLO:** 4 leads at 5%, 2 maintainers at 10% for core, 9-10 maintainers at 5% for 5 subprojects (can have overlap between roles, but need a minimum of 3-4 part-time contributors at 25%)
+- **Feature work:** needs significant ongoing additional investment, minimum of 2-3 FT devs or features will continue to slip
 
-Include any other ways you measure group membership
+### Details
 
-## [Subprojects](https://git.k8s.io/community/sig-instrumentation#subprojects)
+- **KTLO:** Requires two experienced part-time maintainers at 10% (e.g. 2x4h = 8h/wk) in addition to the SIG leadership (chairs/TLs @ minimum of 2h/wk)
+  - Chairs/TLs currently perform the bulk of this work but even amongst the four of them, **do not** have 8h total weekly allocated
+- **Feature work:** requires significantly more investment from development and review time. 
+  - E.g. Structured logging initiative requires a minimum of 2 FT staff for the duration of feature development from beta -> GA
+  - SIG currently owns a number of KEPs stuck in alpha/beta due to lack of dev resources:
+      - [metric cardinality enforcement](https://github.com/kubernetes/enhancements/issues/2305)
 
+### Subprojects
+
+- Subprojects are currently mature/stable and mainly have KTLO needs
+- kube-state-metrics: 2-3 experienced maintainers at 5%, currently staffed
+- Metrics-server: 2 experienced maintainers at 5%, currently staffed
+- Custom-metrics-apiserver: 1 experienced maintainer at 5%, currently staffed
+- Usage-metrics-collector: 4 experienced maintainers at 5%, currently staffed
```

Comment:
isn't 4 experienced maintainers quite a lot for KTLO?

Reply:
What kind of a comment is this? It's not really actionable.